package.json

```diff
@@ -123,6 +123,45 @@
         "command": "io.orta.jest.coverage.toggle",
         "title": "Jest: Toggle Coverage Overlay"
       }
+    ],
+    "debuggers": [
+      {
+        "type": "vscode-jest-tests",
+        "label": "Debug Jest tests using vscode-jest",
+        "languages": ["javascript", "javascriptreact", "typescript", "typescriptreact"],
+        "configurationSnippets": [
+          {
+            "label": "Jest: Default jest configuration",
+            "description": "Running tests by directly running jest",
+            "body": {
+              "type": "node",
+              "name": "vscode-jest-tests",
+              "request": "launch",
+              "program": "${workspaceFolder}/node_modules/jest/bin/jest",
```

Comment:
I tested adding these snippets in launch.json, all of them failed to start debugger... looks like the `${workspaceFolder}` is not populated. Looking at the example from [here](https://code.visualstudio.com/docs/extensions/example-debuggers), you will most likely need to escape them, for example: `"program": "^\"\\${workspaceFolder}/node_modules/jest/bin/jest\""`

were you able to run with these snippets?

Reply:
Ah thank you, since they should have been the same, I actually never tested the snippets from `package.json` except with my eyes; is fixed.
I also extended the test and the readme file.